tests/expectations/compiler/array/array_too_large_fail.out

```diff
@@ -1,10 +1,5 @@
-Error [ETYC0372077]: An array cannot have more than 32 elements, found one with 33 elements
-    --> compiler-test:3:20
-     |
-   3 |     transition foo(a: [bool; 33]) -> bool {
-     |                    ^^^^^^^^^^^^^
-Error [ETYC0372077]: An array cannot have more than 32 elements, found one with 33 elements
-    --> compiler-test:4:19
-     |
-   4 |         let bar = [false; 33];
-     |                   ^^^^^^^^^^^
+program test.aleo;
+
+function foo:
+    input r0 as [boolean; 33u32].private;
```

Comment:
Is the array size limit different now? If so, we should probably change this test to test the new limits.